cime/scripts/lib/CIME/build.py

```diff
@@ -433,7 +440,7 @@ def _create_build_metadata_for_component(config_dir, libroot, bldroot, case):
     In many cases, the bld/configure script will have already created these.
     """
     buildlib = imp.load_source("buildlib_cmake", os.path.join(config_dir, "buildlib_cmake"))
-    buildlib.buildlib(bldroot, libroot, case)
+    return buildlib.buildlib(bldroot, libroot, case)
```

Comment:
We want to support the old behavior, allowing buildlib to return None if that component does need to add any CMake args:

```
cmake_args = buildlib.buildlib(bldroot, libroot, case)
return "" if cmake_args is None else cmake_args
```

Reply:
ah, good point.